code/go/0chain.net/blobbercore/handler/object_operation_handler.go

```diff
@@ -445,28 +445,51 @@ func (fsh *StorageHandler) CommitWrite(ctx context.Context, r *http.Request) (*b
 	elapsedWritePreRedeem := time.Since(startTime) - elapsedAllocation - elapsedGetLock -
 		elapsedGetConnObj - elapsedVerifyWM
 
-	err = connectionObj.ApplyChanges(ctx, writeMarker.AllocationRoot, writeMarker.Timestamp)
+	fileIDMetaStr := r.FormValue("file_id_meta")
+	fileIDMeta := make(map[string]string, 0)
```

Comment:
Usual case will take few hundered bytes.

This map is sent only for upload and copy operation where new uuid is required.
Given the feature that user can make multiple operations and commit with single writemarker, it can be any number.

Currently we have not provided multiple operations in gosdk with same or different connectionID. It only is possible scenario in post production when there shall be different sdk.

Reply:
I'm not sure if this fileIDMeta map is a must, perhaps it can help for debugging when consensus could not meet, otherwise, the fileMetaRoot or the root file ID might be sufficient.

Reply:
Hmm, seems it is needed as if we copy a dir, the file IDs for all the file in the dir will be provided by the file id meta map. We may need to add a limit, otherwise if we have large number of files in a dir, the process could be extremely slow.

Reply:
Just as how I've used uuid to synchronize fileIDs among blobber, we can use same methodology i.e. next uuid generation from given uuid.
With this client shall provide single uuid for blobber to generate other uuids.
Shall I make this change in next PR or in current PR? 
I would prefer to work in new PR.

Reply:
Yes, we can do it in a new PR